src/routes/namespaces/[namespace]/workflows/_workflows-summary-row.svelte

```diff
@@ -2,83 +2,50 @@
   import type { WorkflowExecution } from '$lib/models/workflow-execution';
 
   import { namespace } from '$lib/stores/namespace';
-  import { page } from '$app/stores';
   import { getWorkflowExecutionUrl } from '$lib/utilities/get-workflow-execution-url';
-  import { pathMatches } from '$lib/utilities/path-matches';
   import Time from '$lib/components/workflow-time.svelte';
   import WorkflowStatus from '$lib/components/workflow-status.svelte';
 
   export let workflow: WorkflowExecution;
   export let timeFormat: string;
 
   $: href = getWorkflowExecutionUrl($namespace, workflow);
-  $: isActive = pathMatches(href, $page.path);
 </script>
 
-<tr class:active={isActive}>
-  <td class="">
-    <a sveltekit:noscroll {href}>
+<a sveltekit:noscroll {href}>
```

Comment:
This is so much nice and behaves better than the table rows. The scroll is _way_ smoother now. Nice work.